docs/examples/sock-shop.md

```diff
@@ -141,11 +175,7 @@ namespace/sock-shop created
 Then, we can run:
```

Comment:
Linked to above comment. Following the steps so far, I already have the sock-shop namespace as that was created as part of the kustomize deployment....

Reply:
Surely I need to remove the application before I try and re-add as a helm chart?

Reply:
Ooo... good point. I was always assuming a fresh cluster but never explicitly called that out!

Reply:
Looks like this is already fixed